### DISCUSSION TITLE
litematica menu

### DISCUSSION THREAD

**Discussion by mizuryu7** [maruohon/litematica#646] [Q&A]
I have a problem accidently i unassign the button to open litematica menu. So i can't open litematica. How can i do know ?

**Comment by maruohon** [comment]
Did you unbind just the main menu key `M`or also the direct config menu hotkey `M + C`? You can get to the configs via either. And if you have the Mod Menu mod installed, then you can also get to the config menu via the settings button in the mod list.

Or the final option is to manually edit the config file at `.minecraft/config/litematica.json` and set a hotkey there - you would want to search for `openGuiMainMenu` and `openGuiSettings`.
